docs/api-events.asciidoc

```diff
@@ -88,7 +88,7 @@ An example error response might look something like this:
       "message": "<json-decoding-err>",
       "document": "<ndjson-obj>"
     },{
-      "message": "queue is full" <3>
+      "message": "too many requests" <3>
```

Comment:
Is this a good replacement example for an immediately returning non-event-related error?

Reply:
I think they'll now see "request timed out" but I need to double check the exact message.

Reply:
Is there a better example then? I'm not sure how to move forward here.

Reply:
I'm going to merge and we can address any follow-ups in a new PR.

Reply:
Apologies, I missed your comment regarding an example. In this case the APM Agent will just log a timeout because the APM Server doesn't respond in time. So there is no well defined response in this case. I think it would be best to describe the scenario this way (server not responding in time, logging `request timed out`).